### PR TITLE
[enterprise-4.13] OCPBUGS-19453: add admonition about node reboot

### PR DESCRIPTION
**Pull request by jldohmann**
Version(s): 4.12, 4.13

Issue: https://issues.redhat.com/browse/OCPBUGS-19453

Link to docs preview: https://73711--ocpdocs-pr.netlify.app/openshift-enterprise/latest/security/certificates/replacing-default-ingress-certificate#replacing-default-ingress_replacing-default-ingress

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by jldohmann** [comment]
@sergiordlr could you PTAL for QE ack? this is related to #72185, but for pre-OCP 4.14 behavior where the node drains and reboots after updating certs. this PR adds a small note before the procedure about this

File: modules/customize-certificates-replace-default-router.adoc [hunk 1]
```diff
@@ -25,6 +25,11 @@ followed with any intermediate certificates, and the file should end with the
 root CA certificate.
 * Copy the root CA certificate into an additional PEM format file.
 
+[IMPORTANT]
+====
+Updating the certificate authority (CA) will cause the nodes of your cluster to reboot.
```
Comment (bergerhoffer) [inline modules/customize-certificates-replace-default-router.adoc:30]: Suggest active voice

```suggestion
Updating the certificate authority (CA) causes the nodes in your cluster to reboot.
```

**Review by bergerhoffer — COMMENTED** [review]
Small suggestion!

**Comment by rioliu-rh** [comment]
LGTM

**Pull request merged**